### ISSUE TITLE
Proposal: Show TODO warning if a 'TODO' is in a @doc tag

### ISSUE THREAD

**Issue by dylan-chong** [rrrene/credo#358]
### Precheck

* For proposing a new feature, be sure to name your issue "Proposal: ..."
* For bugs, please do a quick search and make sure the bug has not yet been reported here

### Environment

* Credo version (`mix credo -v`): 0.7.2
* Erlang/Elixir version (`elixir -v`):  1.4.1
* Operating system: Mac OS 10.11.6

### What were you trying to do?

Put a 'TODO' inside a @doc tag

E.g.

```
defmodule X do
  @doc """
  TODO something
  """
  def x do
  end
end
``
### Expected outcome

I should see some "Found a TODO tag in a comment" warnings when I run `mix credo`


### Actual outcome

No TODO warnings show

**Comment by rrrene** [comment]
@dylan-chong I added this yesterday and just pushed it to `master`. Thx for the great suggestion! :+1:

**Comment by dylan-chong** [comment]
Nice! Should this issue be closed then?

**Comment by rrrene** [comment]
I will it once the patch is part of a release!
